cypress/plugins/index.js

```diff
@@ -1,19 +0,0 @@
-// ***********************************************************
-// This example plugins/index.js can be used to load plugins
-//
-// You can change the location of this file or turn off loading
-// the plugins file with the 'pluginsFile' configuration option.
-//
-// You can read more here:
-// https://on.cypress.io/plugins-guide
-// ***********************************************************
-
-// This function is called when a project is opened or re-opened (e.g. due to
-// the project's config changing)
-let percyHealthCheck = require("@percy/cypress/task");
-
-module.exports = (on, config) => {
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
-  on("task", percyHealthCheck);
```

Comment:
Instead of deleting the whole file, can you just remove this line & 13?

Reply:
Yes sure, so then it will be the same as the default cypress config.